hopsworks-api/src/main/java/io/hops/hopsworks/api/admin/dto/VariablesRequest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hops.hopsworks.api.admin.dto;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+@XmlRootElement
+public class VariablesRequest {
```

Comment:
We should think about a good naming scheme for these request objects. Previously in the code they have been called blablaDTO (data transfer object). I call them by name in my code, so "Variables", or "VariableView" for "views". Actually, I think the most reasonable thing is to call them just by the entity name: "Variables". What do you guys think?

Reply:
Until we conclude to a naming scheme, I leave it as it is.